.travis.yml

```diff
@@ -1,15 +1,20 @@
-language: c
-sudo: false
+language: c++
+# Need sudo for trusty.
+sudo: required
+# Trusty gives us GCC 4.8 which supports c++11.
+dist: trusty
```

Comment:
Ya I was playing with that when I last worked on this. I couldn't seem to get it to pass validation (http://lint.travis-ci.org/) so I was trying to figure out if I did something wrong. I will just try committing it to this PR and see if it works.